packages/blade/src/components/Button/Button/Button.stories.tsx

```diff
@@ -75,6 +78,7 @@ export default {
   },
   argTypes: {
     ...getStyledPropsArgTypes(),
+    ...getCommonEventArgTypes(),
```

Comment:
The story table is becoming gigantic now 😅

Reply:
Yeah agree, can check later if it's possible to make unneeded panels collapsed by default.